cmd/cmd_test.go

```diff
@@ -279,6 +279,9 @@ func TestSaveTwoComponents(t *testing.T) {
 
 	// Save a version, then same another with two components at once
 	run.MustExec(t, "qri save --file=testdata/movies/ds_ten.yaml me/test_ds")
+	got1 := run.MustLoadDataset(t, run.GetPathForDataset(t, 0))
+	fmt.Println(got1.Stats.Path)
```

Comment:
Is this print needed? Put another way, is it always going to be the same value, and can we turn it into an assertion instead?